deployment/CP05MOAS-GL387_Deploy.csv

```diff
@@ -29,15 +29,21 @@ SI20170418,,TS20170702,,CP05MOAS-GL387-02-FLORTM000,5,1,2017-04-19T20:53:02,2017
+AR24,,TS20180103,,CP05MOAS-GL387-01-ADCPAM000,6,1,2017-10-28T22:02:00,2018-01-03T19:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00387,CGINS-ADCPAM-49697,40.03183333,-70.62041667,,1000,1000,FZ-2
+AR24,,TS20180103,,CP05MOAS-GL387-02-FLORTM000,6,1,2017-10-28T22:02:00,2018-01-03T19:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00387,CGINS-FLORTM-03116,40.03183333,-70.62041667,,1000,1000,FZ-2
+AR24,,TS20180103,,CP05MOAS-GL387-03-CTDGVM000,6,1,2017-10-28T22:02:00,2018-01-03T19:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00387,CGINS-CTDGVM-09089,40.03183333,-70.62041667,,1000,1000,FZ-2
+AR24,,TS20180103,,CP05MOAS-GL387-04-DOSTAM000,6,1,2017-10-28T22:02:00,2018-01-03T19:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00387,CGINS-DOSTAM-00604,40.03183333,-70.62041667,,1000,1000,FZ-2
+AR24,,TS20180103,,CP05MOAS-GL387-05-PARADM000,6,1,2017-10-28T22:02:00,2018-01-03T19:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00387,CGINS-PARADM-50167,40.03183333,-70.62041667,,1000,1000,FZ-2
+AR34,,WJ20190617,,CP05MOAS-GL387-00-ENG000000,7,1,2019-04-08T01:35:00,2019-06-18T10:34:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00387,CGCON-GLDRCP-00387,39.9681,-71.00002,,1000,1000,FZ-1
+AR34,,WJ20190617,,CP05MOAS-GL387-01-ADCPAM000,7,1,2019-04-08T01:35:00,2019-06-18T10:34:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00387,CGINS-ADCPAM-72777,39.9681,-71.00002,,1000,1000,FZ-1
+AR34,,WJ20190617,,CP05MOAS-GL387-02-FLORTM000,7,1,2019-04-08T01:35:00,2019-06-18T10:34:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00387,CGINS-FLORTM-04465,39.9681,-71.00002,,1000,1000,FZ-1
+AR34,,WJ20190617,,CP05MOAS-GL387-03-CTDGVM000,7,1,2019-04-08T01:35:00,2019-06-18T10:34:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00387,CGINS-CTDGVM-09027,39.9681,-71.00002,,1000,1000,FZ-1
+AR34,,WJ20190617,,CP05MOAS-GL387-04-DOSTAM000,7,1,2019-04-08T01:35:00,2019-06-18T10:34:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00387,CGINS-DOSTAM-00569,39.9681,-71.00002,,1000,1000,FZ-1
+AR34,,WJ20190617,,CP05MOAS-GL387-05-PARADM000,7,1,2019-04-08T01:35:00,2019-06-18T10:34:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00387,CGINS-PARADM-50146,39.9681,-71.00002,,1000,1000,FZ-1
+TS20201121,,,,CP05MOAS-GL387-00-ENG000000,8,1,2020-11-21T17:40:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00387,CGCON-GLDRCP-00387,39.88723,-70.35692,,1000,1000,FZ-1
+TS20201121,,,,CP05MOAS-GL387-01-ADCPAM000,8,1,2020-11-21T17:40:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00387,CGINS-ADCPAM-55979,39.88723,-70.35692,,1000,1000,FZ-1
+TS20201121,,,,CP05MOAS-GL387-02-FLORTM000,8,1,2020-11-21T17:40:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00387,CGINS-FLORTM-03884,39.88723,-70.35692,,1000,1000,FZ-1
+TS20201121,,,,CP05MOAS-GL387-03-CTDGVM000,8,1,2020-11-21T17:40:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00387,CGINS-CTDGVM-09055,39.88723,-70.35692,,1000,1000,FZ-1
+TS20201121,,,,CP05MOAS-GL387-04-DOSTAM000,8,1,2020-11-21T17:40:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00387,CGINS-DOSTAM-00145,39.88723,-70.35692,,1000,1000,FZ-1
+TS20201121,,,,CP05MOAS-GL387-05-PARADM000,8,1,2020-11-21T17:40:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00387,CGINS-PARADM-50324,39.88723,-70.35692,,1000,1000,FZ-1

```

Comment:
New deployment looks good.  CUID updates look good.  For glider 387 the PAR sensor SN needs to be corrected on FCP from 50234 to 50324 (confirmed that 50324 is correct via vendor cal doc and autoexec file in FCP zip folder).